projects/packages/account-protection/src/class-settings.php

```diff
@@ -30,15 +30,31 @@ public function __construct( ?Account_Protection $account_protection = null ) {
 	/**
 	 * Get account protection settings.
 	 *
-	 * @return array
+	 * @return array Settings.
 	 */
-	public function get() {
+	public function get(): array {
 		$settings = array(
 			'isEnabled'                    => $this->account_protection->is_enabled(),
 			'isSupported'                  => $this->account_protection->is_supported_environment(),
 			'hasUnsupportedJetpackVersion' => $this->account_protection->has_unsupported_jetpack_version(),
+			'config'                       => $this->get_config(),
 		);
 
 		return $settings;
 	}
+
+	/**
+	 * Get account protection config.
+	 *
+	 * @return array Config settings.
+	 */
+	public function get_config(): array {
+		$advanced_options = $this->account_protection->environment_supports_advanced_options();
+
+		return array(
+			'supports_advanced_options'                    => $advanced_options,
+			Config::PASSWORD_DETECTION_ENABLED_OPTION_NAME => (bool) get_option( Config::PASSWORD_DETECTION_ENABLED_OPTION_NAME, ! $advanced_options ),
+			Config::STRONG_PASSWORDS_ENABLED_OPTION_NAME   => (bool) get_option( Config::STRONG_PASSWORDS_ENABLED_OPTION_NAME, ! $advanced_options ),
```

Comment:
I don't think we need to use the option names as keys here, I know we do this for the WAF, however within the context of this internal API we don't need the prefix, we don't actually want the API schema to change if the option names used in the database change, and the settings API class is currently using snakeCase and this introduces mixed casing in the property names.